junit-platform-commons/src/main/java/org/junit/platform/commons/util/RuntimeUtils.java

```diff
@@ -40,16 +40,7 @@ private RuntimeUtils() {
 	 * Try to determine whether the VM was started in debug mode or not.
 	 */
 	public static boolean isDebugMode() {
-		Optional<List<String>> optionalArguments = getInputArguments();
-		if (!optionalArguments.isPresent()) {
-			return false;
-		}
-		for (String argument : optionalArguments.get()) {
-			if (argument.startsWith("-agentlib:jdwp")) {
-				return true;
-			}
-		}
-		return false;
+		return getInputArguments().map(args -> args.stream().anyMatch(arg -> arg.contains("jdwp"))).orElse(false);
```

Comment:
❌ I think we shouldn't be so lenient and rather check for `argument.startsWith("-agentlib:jdwp") || argument.startsWith("-Xrunjdwp")` (see related [StackOverflow post](https://stackoverflow.com/a/173447/6327046)).

Reply:
Sure thing. Done :)